WooCommerce/Classes/POS/Controllers/PointOfSaleCouponsController.swift

```diff
@@ -100,6 +100,16 @@ private extension PointOfSaleCouponsController {
                 setCouponsLoadedViewState(storedCoupons, hasMoreItems: true)
             }
 
+            await refreshFirstPage()
+        } catch {
+            if let couponError = error as? PointOfSaleCouponServiceError {
+                setCouponsErrorViewState(couponError)
+            }
+        }
```

Comment:
Are we sure we want to silently ignore any other errors?
```suggestion
        } catch let error as PointOfSaleCouponServiceError {
            setCouponsErrorViewState(error)
        } catch {
            /// Other error handling/logging
        }
```